SimGeneral/MixingModule/python/pixelDigitizer_cfi.py

```diff
@@ -22,6 +22,15 @@
 )
 
 from SimTracker.SiPhase2Digitizer.phase2TrackerDigitizer_cfi import phase2TrackerDigitizer as _phase2TrackerDigitizer
+
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase2_tracker.toReplaceWith(pixelDigitizer, _phase2TrackerDigitizer)
 
+# tracker phase2 post-TDR
+from Configuration.StandardSequences.Eras import eras
+eras.phase2_tracker_postTDR.toModify(_phase2TrackerDigitizer, PixelDigitizerAlgorithm  = dict(TanLorentzAnglePerTesla_Barrel = 0.0459))
+eras.phase2_tracker_postTDR.toModify(_phase2TrackerDigitizer, PixelDigitizerAlgorithm  = dict(TanLorentzAnglePerTesla_Endcap = 0.0459))
```

Comment:
the change in this line can be combined with the previous line (multiple items can be specified within `dict()`).

Reply:
Assuming `phase2_tracker_postTDR` will always require `phase2_tracker`, these customizations should be moved to 
https://github.com/cms-sw/cmssw/blob/78e25c56c86bae6d05ba616e797d5ec67a13e321/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py#L174
(before the premixing customization). Then this file would not need any changes (`phase2_tracker` already replaces `pixelDigitizer` with `_phase2TrackerDigitizer`), and the customizations will work automatically correctly with the upcoming premixing.